test/src/test/java/com/distkv/dst/supplier/MasterSalverTestUtil.java

```diff
@@ -0,0 +1,94 @@
+package com.distkv.dst.supplier;
+
+import com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class MasterSalverTestUtil {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MasterSalverTestUtil.class);
+
+  private static final String SERVER_SUFFIX_JAR_DIR = "server" + File.separator + "target"
+      + File.separator + "dst-server-0.1.3-SNAPSHOT-jar-with-dependencies.jar";
+  private static final int NODE_NUM = 3;
+
+  private static Process[] processes = new Process[NODE_NUM];
+
+  private static final int KILL_PROCESS_WAIT_TIMEOUT_SECONDS = 1;
+
+  /**
+   * @param command The command that will be executed.
+   */
+  private static Process executeCommand(List<String> command) {
```

Comment:
No need to write so many duplicated code.

Reply:
I can send another PR later to fix this problem.

Reply:
Please open an issue for trace this, thanks.

Reply:
OK, #431